proto/warden/vemanager/vemanager.proto

```
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package warden.vemanager;
```

buf (1.47.2)

Files with package "warden.vemanager" must be within a directory "warden/vemanager" relative to root but were in directory "proto/warden/vemanager".

(PACKAGE_DIRECTORY_MATCH)